nemo/core/optim/optimizers.py

```diff
@@ -51,7 +51,6 @@
     AVAILABLE_OPTIMIZERS['fused_adam'] = FusedAdam
 except ModuleNotFoundError:
     HAVE_APEX = False
-    logging.warning("Apex was not found. Using the lamb or fused_adam optimizer will error out.")
```

Comment:
Thanks for removing this.. I forgot to send a PR for it.

Reply:
Lol forgot myself as well~